zulipterminal/helper.py

```diff
@@ -39,6 +39,7 @@
     all_starred=set(),
 )
 
+unindexed_messages = dict()  # type: Dict[int, Any]
```

Comment:
I understand why you might want to avoid having this in the Index itself, but I'm unconvinced it should be a global variable.

For now, placing this in the model might be simpler?